src/replicache.ts

```diff
@@ -27,6 +27,9 @@ export const httpStatusUnauthorized = 401;
 
 type MaybePromise<T> = T | Promise<T>;
 
+/** The key name to use in localStorage when synchronizing changes. */
+const storageKeyName = (name: string) => `/replicache/${name}`;
```

Comment:
Nit: perhaps this should be configurable by user of API in off chance it collides somehow?

Reply:
Punting on this. Seems like we can change later if there is ever a case. I can change the key name to `/replicache-root/${name}` to make it even less likely to have conflicts?